<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFESpectraViewerDetectorEditor</class>
 <widget class="QDialog" name="QFESpectraViewerDetectorEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Lightsource</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFEnhancedLineEdit" name="edtID"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>name:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFEnhancedLineEdit" name="edtName"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>order no.</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFEnhancedLineEdit" name="edtOrder"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>manufacturer:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QFEnhancedLineEdit" name="edtManufacturer"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>reference:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QFEnhancedLineEdit" name="edtReference"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>peak wavelength:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="spinWavelength">
         <property name="suffix">
          <string> nm</string>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>description:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QFEnhancedLineEdit" name="edtDescription"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>peak sensitivity:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1,0">
         <item>
          <widget class="QDoubleSpinBox" name="spinSensitivity">
           <property name="suffix">
            <string> nm</string>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>unit:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFEnhancedLineEdit" name="edtUnit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>/W</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>folder:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFEnhancedLineEdit" name="edtFolder"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Spectrum</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnLoadSpectrumFile">
           <property name="text">
            <string>load from &amp;file</string>
           </property>
           <property name="icon">
            <iconset resource="../../lib/lib.qrc">
             <normaloff>:/lib/file_opendir.png</normaloff>:/lib/file_opendir.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPasteSpectrum">
           <property name="text">
            <string>&amp;paste</string>
           </property>
           <property name="icon">
            <iconset resource="../../lib/lib.qrc">
             <normaloff>:/lib/edit_paste.png</normaloff>:/lib/edit_paste.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteRow">
           <property name="text">
            <string>delete row</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteColumn">
           <property name="text">
            <string>delete &amp;column</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearSpectrum">
           <property name="text">
            <string>&amp;clear spectrum</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QEnhancedTableView" name="tableView">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>reference:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFEnhancedLineEdit" name="edtSpectrumReference"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEnhancedTableView</class>
   <extends>QTableView</extends>
   <header>qenhancedtableview.h</header>
  </customwidget>
  <customwidget>
   <class>QFEnhancedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfenhancedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtID</tabstop>
  <tabstop>edtFolder</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtOrder</tabstop>
  <tabstop>edtManufacturer</tabstop>
  <tabstop>edtReference</tabstop>
  <tabstop>spinWavelength</tabstop>
  <tabstop>spinSensitivity</tabstop>
  <tabstop>edtUnit</tabstop>
  <tabstop>edtDescription</tabstop>
  <tabstop>btnLoadSpectrumFile</tabstop>
  <tabstop>btnPasteSpectrum</tabstop>
  <tabstop>btnDeleteRow</tabstop>
  <tabstop>btnDeleteColumn</tabstop>
  <tabstop>btnClearSpectrum</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>edtSpectrumReference</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
  <include location="../../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFESpectraViewerDetectorEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFESpectraViewerDetectorEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
